__mocks__/react-native.mock.js

```
@@ -23,6 +23,13 @@ jest.mock('react-native', () => {
         decryptData: input => (input ? String(input) : 'mockedString'),
         deviceSupportsHardware: () => true,
       },
+      InjiVciClient: {
+        addListener: jest.fn(),
```

ESLint

[error] 'jest' is not defined.

(no-undef)